examples/flow_tracker_subscriber/README.md

```diff
@@ -0,0 +1,31 @@
+Simple Forward
```

Comment:
for now just change the title here to `Flow Tracker NF` but later we should provide documentation.